.github/workflows/yjit-ubuntu.yml

```diff
@@ -21,9 +21,9 @@ jobs:
 #         - ubuntu-18.04
         yjit_opts: [
           "--yjit",
-          "--yjit --yjit-call-threshold=1 --yjit-max-versions=1",
-          # "--yjit --yjit-call-threshold=1",
```

Comment:
I think @nobu disabled these because we had too many runs and was slowing down CI. I'm not sure if it's okay to add these back?

Reply:
Hm. Ok. I wish they would tell us before doing things like that. I can leave just two enabled.